### PR TITLE
Cleanup. Removed unused vars and imports

### PR DESCRIPTION
**Pull request by prabhu**

### CODE REVIEW

File: lib/collection.js [hunk 1]
```diff
@@ -222,13 +218,7 @@ Collection.prototype.isCapped = function(cb) {
   });
 };
 
-Collection.prototype.stats = function(cb) {
```
Comment (sorribas) [inline lib/collection.js]: Why remove this method? This is a public method documented here. http://docs.mongodb.org/manual/reference/method/db.collection.stats/#db.collection.stats

**Comment by sorribas** [comment]
Hey @prabhu thanks so much for this PR. Other than the comment that I made on the source looks good to merge.

File: lib/collection.js (hunk 1 above)
Comment (prabhu) [inline lib/collection.js]: The same method appears twice in the source. Probably due to an earlier merge.

Comment (sorribas) [inline lib/collection.js]: Oh yeah! You're right.

**Comment by sorribas** [comment]
Thanks!

**Pull request merged**